### PR TITLE
DE: add example vaccination certificates

### PR DESCRIPTION
**Pull request by SZoerner**
These are three vaccination certificates: the first two are valid DGCs issued by our EU Gateway DSC authorities (acceptance stage). The third one is issued by an authority not part of our Trusted List of DSCs.

### CODE REVIEW

File: DE/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Test files
+
+### 1 - Tina Testerin
+
+[1.json](2DCode/raw/1.json) - Basic DGC with one vaccination entry. Signed with the first DSC.
+
+All tests should be successful.
+
+![1](png/1.png)
```
Comment (SZoerner) [inline DE/README.md:13]:
```suggestion
![1](2DCode/png/1.png)
```

File: DE/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### 2 - Erika Musterfrau
+
+[2.json](2DCode/raw/2.json) - Basic DGC with one vaccination entry. Signed with a second DSC.
+
+All tests should be successful.
+
+![2](png/2.png)
```
Comment (SZoerner) [inline DE/README.md:21]:
```suggestion
![2](2DCode/png/2.png)
```

File: DE/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### 3 - Max Mustermann
+
+[3.json](2DCode/raw/3.json) - Basic DGC with one vaccination entry. Signed with a third key, not the DSC indicated.
+
+Signature validation should fail.
+
+![3](png/3.png)
```
Comment (SZoerner) [inline DE/README.md:29]:
```suggestion
![3](2DCode/png/3.png)
```

**Comment by JChrist** [comment]
The `kid` contained in the certificate(s) is invalid/does not match the `kid` as calculated from the certificate provided in the test data.
In particular, for `1.json`, the certificate contains `kid` (in base64 for easy reference): `jt4zFtTaQYGB8HU6/8ajow==`
The expected one - based on the CERTIFICATE field in the test data - would be: `DEsVUSvpFAE=`
The same error exists for other provided certificates as well.

**Review by SchulzeStTSI — APPROVED** [review]

**Pull request merged**